source/addonHandler/__init__.py

```diff
@@ -401,7 +402,7 @@ def isRunning(self):
 
 	@property
 	def isDisabled(self):
-		return self.name in _disabledAddons
+		return globalVars.appArgs.disableAddons or self.name in _disabledAddons

```

Comment:
Rather than modify `isDisabled`, which reflects the user preference state, add a condition to `isRunning`